contxt/legacy/services/ems.py

```diff
@@ -0,0 +1,102 @@
+from datetime import datetime
+from typing import Optional
+
+from contxt.legacy.models.ems import (
+    FacilityMainService,
+    FacilityUtilitySpend,
+    FacilityUtilityUsage,
+)
+from contxt.legacy.services import GET, APIObjectCollection, Service
+
+CONFIGS_BY_ENVIRONMENT = {
```

Comment:
yea i should have specified, but all the `legacy/*` files are the old services. the goal of this pr was to replace them, and ideally they should but removed. however, i've just decided to keep them around temporarily because other parts of this project rely on their current form and i wanted to make this pr more focused